CHANGELOG.md

```diff
@@ -3,6 +3,118 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.19.0
+
+_Oct 4, 2024_
+
+We'd like to offer a big thanks to the 26 contributors who made this release possible. Here are some highlights ✨:
+
+- ✅ Automatic parents and children selection in the Data Grid Premium component
```

Comment:
Might relate it to the features rather than package?
```suggestion
- 🔁 Automatic parents and children selection for Data Grid "tree data" and "row grouping" features
```